### PR TITLE
MAINT use variables for Python version rather than hardcoding it

### PR DESCRIPTION
**Pull request by hoodmane**
This should make Python updates easier.

### CODE REVIEW

File: pyodide-build/pyodide_build/testing.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import contextlib
 import inspect
 from base64 import b64encode
+from sys import version_info
 from typing import Callable, Collection, Optional
 
 import pytest
 
+PYVERSION = f"python{version_info.major}.{version_info.minor}"
```
Comment (ryanking13) [inline pyodide-build/pyodide_build/testing.py:9]: I think tests should use Pyodide Python version not local Python version, though currently our assumption is that those two are strictly same.

Comment (hoodmane) [reply pyodide-build/pyodide_build/testing.py:9]: Yeah. This is easier to write though. What should I do to get the Pyodide version here?

Comment (ryanking13) [reply pyodide-build/pyodide_build/testing.py:9]: Well I think tests that require knowing python version can just do `version = pyodide.runPython('from sys import version_info, ...')` each time like you did at the `pyodide.ts`.
If there are too many of those, maybe we could expose a version variable to js such as `pyodide.pythonVersion` (I'm not sure though)

Comment (hoodmane) [reply pyodide-build/pyodide_build/testing.py:9]: We could just add it as a attribute to selenium e.g., `selenium.pyversion`

Comment (hoodmane) [reply pyodide-build/pyodide_build/testing.py]: Okay, how about this?

Comment (ryanking13) [reply pyodide-build/pyodide_build/testing.py]: Yeah, this looks better. In the long term, I think we should split out testing-related things from pyodide-build, but that could be discussed in other place.

**Review by ryanking13 — APPROVED** [review]
Thanks, @hoodmane. This would indeed make updating Python version easier.

Do you have plans to also update hardcoded Python version strings in packages directory in this PR? (e.g. [scipy](https://github.com/pyodide/pyodide/blob/a725c6ce0fbd145c5ebc66f50710197dbb0f216d/pac kages/scipy/meta.yaml#L92)).

**Comment by hoodmane** [comment]
I currently have that on #2238.

**Comment by hoodmane** [comment]
So I'm good with merging this as is and dealing with more version numbers soon in other prs.

**Pull request merged**